lucille-examples/lucille-distributed-example/README.md

```diff
@@ -0,0 +1,14 @@
+## How to run the example
+
+1. Run ```mvn clean install -DskipTests``` at the top level directory of lucille to build the project.
+2. Run ```mvn verify -Pnightly``` at the top in the ```lucille-distributed-example``` directory to run the distributed example.
+
+### What the mvn verify -Pnightly command does
+
+1. It runs the nightly profile defined in the pom.xml. It runs an exec(docker compose up with --abort-on-container-exit), runs a test within the runner container and exits with the exit code of the runner container.
+2. The docker-compose.yaml spins up lucille in distributed mode, and runs 2 different lucille runs.
```

Comment:
in lucille terminology, there's only one "Run" here (with a single runID). There are two separate connectors that are executed sequentially in this run.

Reply:
updated README in [9a9588d](https://github.com/kmwtechnology/lucille/pull/206/commits/9a9588d35d1692965ad82c61682f471f9ade6dfe)